app/view_objects/find/results_view.rb

```diff
@@ -27,12 +27,15 @@ def query_parameters_with_defaults
     end
 
     def courses
-      @courses ||= RecruitmentCycle.current.courses.includes(
-        :enrichments,
-        subjects: [:financial_incentive],
-        site_statuses: [:site],
-        provider: %i[recruitment_cycle ucas_preferences],
-      ).findable.page(query_parameters[:page] || 1)
+      @courses ||= Find::CourseSearchService.call(
+        filter: query_parameters,
+        sort: query_parameters[:sortby],
+        course_scope: recruitment_cycle.courses,
```

Comment:
This is incorrect, the scope needs to be findable courses, eg https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/controllers/api/v3/courses_controller.rb#L52. This is the scope that is passed into the current endpoint Find talks to so it needs to match. At the moment the review app returns rolled over courses too